src/popup.css

```
@@ -188,4 +188,16 @@ button:hover {
   .ext.type-admin {
     color: #ffa38d;
   }
+}
+
+/**
+ * Right to Left
+ */
+html.rtl {
+  direction: rtl;
+  text-align: right;
+}
+
+html.rtl button {
+  text-align: right;
 }
```

```suggestion
html[dir='rtl'] {
  text-align: right;
}

html[dir='rtl'] button {
  text-align: right;
}
```

I think the `dir` attribute also automatically sets the direction. Maybe it sets _all_ of this. Can you check if it works without this CSS?